apps/filecoin-site/src/app/_components/Navigation/NavigationLink.tsx

```diff
@@ -20,13 +20,13 @@ const MOBILE_TOUCH_TARGET: TouchTarget = {
 }
 
 const desktopStyle = clsx(
-  'navigation-link focus:brand-outline inline-block font-semibold',
+  'navigation-link focus:brand-outline inline-block',
```

Comment:
The main purpose of `globals.css` was to be able to have different styles across libraries for shared UI components. Then, with, filecoin-site, we also started to use it for gradient styles and to differentiate between dark and light themes. We should not however use if for any style, like `font-display`, `tracking-wider`, etc. These should be inlined.